automatic/gimp/update.ps1

```diff
@@ -18,11 +18,10 @@ function global:au_GetLatest {
   $re        = '\.exe$'
   $url32     = $download_page.Links | ? href -match $re | select -first 1 -expand href | % { 'https:' + $_ }
 
-  $verRe     = '[-](?:setup(?:\-[\d]*)?)?|\.exe$'
-  $version32 = $url32 -split "$verRe" | select -last 1 -skip 2
+  $version32 = ((($url32 -split("\/"))[-1]) -replace("[putesmigx]","") ) -replace("\-\-|\-",".")
```

Comment:
Hmm, didn't know that `-1` for the array was a thing in PowerShell. Guess I learn something new every day 😃.

Not sure I like the idea of removing all of the specific characters like you do in the `-replace` option, could become a problem if they ever push out a pre-release in the future (even if it would be by mistake).

Reply:
Thx god for -1

Reply:
@majkinetor @AdmiringWorm 
I'll look to adjust the regex to be a little more lax.